CONTRIBUTION-zh.md

```diff
@@ -0,0 +1,80 @@
+# Contribution (Draft)
+
+Awesome Satori 是用于展示 Satori protocol 社区中各种有趣主题的资源列表，旨在收集和分享与 Satori 相关优秀资源，包括协议实现、SDK、Framework、教程、工具等，帮助其他人学习、发现和使用 Satori。
+为了保证质量和一致性，本篇文档将详细描述您应该如何贡献作品。
```

Comment:
Since it is ugly to have spaces between Chinese characters, we are not using one single new-line here. Instead, you could consider putting them in the same line, or, add an extra new-line to split them into two lines.